packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -240,6 +240,17 @@ const VirtualListBaseFactory = (type) => {
 			} else if (this.hasDataSizeChanged) {
 				this.updateStatesAndBounds(nextProps);
 				this.setContainerSize();
+			} else if (rtl !== nextProps.rtl) {
+				const
+					{scrollPosition} = this,
+					{x, y} = this.isPrimaryDirectionVertical ? {x: 0, y: scrollPosition} : {x: scrollPosition, y: 0};
```

Comment:
I think we can make a function for getting x, y.
`composeStyle` also uses this logic.